src/client/packages/idom-client-react/src/element-utils.js

```diff
@@ -64,7 +64,7 @@ function normalizeAttribute([key, value]) {
     key.startsWith("aria_") ||
     DASHED_HTML_ATTRS.includes(key)
   ) {
-    normKey = key.replace("_", "-");
```

Comment:
Why doesn't this work though

Reply:
An alternative to splitting the string is using `replaceAll`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replaceAll

Reply:
Dang stack overflow lied to me. Will switch to that.

Reply:
To be fair, replaceAll was standardized fairly recently (mid-2020).

Reply:
For compatibility purposes, it might make sense to keep using `key.split("_").join("-")`